components/03-modules/brief-info/brief-info.hbs

```diff
@@ -2,19 +2,21 @@
     <{{heading.tag}} class="brief-info__heading {{heading.class}}">
        {{heading.text}}
     </{{heading.tag}}>
-    {{#each items as |item|}}
-    <div class="brief-info__item">
-        <span class="brief-info__icon-wrapper {{ item.iconWrapperClass }}">
-            {{ render '@icon' item.icon }}
-        </span>
-        <div class="brief-info__content">
-            <{{ item.title.tag }}
-                {{{ item.title.attributes }}}
-                class="brief-info__title {{ item.title.class }}"
-            >
-               {{ item.title.text }}
-            </{{ item.title.tag }}>
-        </div>
+    <div class="brief-info__items-container">
+        {{#each items as |item|}}
+            <div class="brief-info__item">
```

Comment:
maybe we can put it in list? b/c in fact we list here some features...

Reply:
Done